### ISSUE TITLE
Time error

### ISSUE THREAD

**Issue by MaxiTalenti** [anshooarora/extentreports#237]
Hi,

To update a html file with another test, the execution times of the test are modified by values that do not correspond. Times are increasing more and more.

**Comment by anshooarora** [comment]
Fixed.  Will make the other fixes and share the DLL.  Will you be able to retest?

**Comment by MaxiTalenti** [comment]
Yes, of course

**Comment by anshooarora** [comment]
Here it is:  http://relevantcodes.com/Tools/ExtentReports2/extentreports-net-v2.40.1-beta2-build.zip

Also includes fix for #236.

You can specify the locale such as:

```
new ExtentReports("file-path-location",  CultureInfo.GetCultureInfo("es-ES"));
new ExtentReports("file-path-location",  CultureInfo.GetCultureInfo("es-ES"), false);
new ExtentReports("file-path-location",  CultureInfo.GetCultureInfo("es-ES"), false, DisplayOrder.OldestFirst);
```

**Comment by MaxiTalenti** [comment]
Now the time is right, but I still can use the framework in Spanish

**Comment by anshooarora** [comment]
Its still working? Or still not working?

**Comment by anshooarora** [comment]
Closing ticket as the time issue is fixed in c543ecc98fdfcd758842548563bd5df4e79cfb08
